api/models/championship.js

```diff
@@ -4,12 +4,12 @@ var Schema   = Mongoose.Schema;
 var championshipSchema = new Schema({
 	month: Number,
 	year: Number,
-	players: [{type: Schema.Types.ObjectId, ref: 'Player'}],
-	matches: [{type: Schema.Types.ObjectId, ref: 'Match'}],
+	players: [String],
+	matches: [String],
 	date: Date,
 	finalMatch: {type: Schema.Types.ObjectId, ref: 'Match'},
 	isCurrent: Boolean
 });
 
 var Championship = Mongoose.model('Championship', championshipSchema);
-module.exports = Championship;
```

Comment:
why do you change this? wouldn't be better to have the reference to the players and matches instead of only the IDs?